CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 CHANGELOG
 =========
 
+v1.9.0
+------
+* Remove Redis TOI dump command
+* Prune TOI adapted to postgres and standalone tileserver
```

Comment:
Something more like? 

`- Add additional postgres support to tiles-of-interest (TOI) gardening`